lib/config/stylistic.js

```diff
@@ -3,7 +3,11 @@
 module.exports = {
   rules: {
     'block-indentation': true,
+    'eol-last': 'always',
     'linebreak-style': true,
+    'no-multiple-empty-lines': true,
+    'no-trailing-dot-in-path-expression': true,
```

Comment:
This rule should be removed from ember-template-lint, it was added to support a thing that is actually not possible (`{{some.path.here. whoops}}`) any more. IIRC, the guts of the rule are empty and it essentially does nothing.

Reply:
Got it, I'll remove that from the config, and I'll open up a separate PR to remove that rule, if you're interested.